tensorflow/compiler/tf2tensorrt/convert/ops/layer_utils.h

```diff
@@ -295,6 +295,20 @@ class TRTNetworkBuilder {
     return const_layer;
   };
 
+  template <typename T>
+  StatusOr<nvinfer1::IConstantLayer*> Constant(
```

Comment:
Nice! Can we make the overloading version in line 278 to call this routine, to avoid code duplication?

Reply:
Good idea! Done